packages/familie-visittkort/src/index.tsx

```diff
@@ -1,42 +1,27 @@
-import { FamilieIkonVelger } from '@navikt/familie-ikoner';
-import { kjønnType } from '@navikt/familie-typer';
 import * as React from 'react';
 import styled from 'styled-components';
-import { CopyButton, Label } from '@navikt/ds-react';
-import { AGray700, ASpacing2, ASpacing4 } from '@navikt/ds-tokens/dist/tokens';
-
-export interface IkonProps {
-    className?: string;
-    heigth?: number;
-    width?: number;
-}
-export interface IProps {
+import { CopyButton, HStack, Label } from '@navikt/ds-react';
+import { ABorderStrong, ABorderSubtle, ASpacing4 } from '@navikt/ds-tokens/dist/tokens';
+import { FamilieIkonVelger } from '@navikt/familie-ikoner';
+import { kjønnType } from '@navikt/familie-typer';
+export interface IProps extends React.PropsWithChildren {
     alder: number;
     ident: string;
     kjønn: kjønnType;
     navn: string | React.ReactNode;
-    ValgfrittIkon?: React.ComponentType<IkonProps>;
-    children?: React.ReactNode;
+    ikon?: React.ReactElement;
+    dempetKantlinje?: boolean;
+    padding?: boolean;
```

Comment:
Skal man like greit få mulighet til å sende inn sine egne farger og paddinger? Dersom man ikke sender inn egne verdier kan vi bruke default, som før. 

Vet ikke hvor mye man kommer til å trikse og mikse med egne verdier, men samtidig vil jo komponenten bli mer fleksibel

Reply:
Farger: vi gikk for å bruke de semantiske fargene fra Aksel som er anbefalt for borders - disse stemmer også overens med de som brukes i dag :) ba-sak bruker den dempede gråfargen (gamle `@navGra20`)

Egne verdier: ja, vurderte det! Gikk for å holde det enkelt og bruke eksisterende verdier som er i bruk i dag og heller legge det til som en feature hvis vi trenger det i fremtiden 🤔